Source/WebCore/html/HTMLInputElement.cpp

```diff
@@ -780,9 +782,12 @@ void HTMLInputElement::attributeChanged(const QualifiedName& name, const AtomStr
         addToRadioButtonGroup();
         HTMLTextFormControlElement::attributeChanged(name, oldValue, newValue, attributeModificationReason);
         break;
-    case AttributeNames::checkedAttr:
-        if (m_inputType->isCheckable())
+    case AttributeNames::checkedAttr: {
+        std::optional<Style::PseudoClassChangeInvalidation> checkedInvalidation;
+        if (m_inputType->isCheckable()) {
             invalidateStyleForSubtree();
```

Comment:
If PseudoClassChangeInvalidation is sufficient, `invalidateStyleForSubtree` can be removed.

Reply:
My change here was not necessary in the first place; we already invalidate PseudoClassType::Checked in setChecked.